code/_compile_options.dm

```diff
@@ -84,9 +84,12 @@
 /// If this is uncommented, will profile mapload atom initializations
 // #define PROFILE_MAPLOAD_INIT_ATOM
 
-/// If uncommented, Dreamluau will be fully disabled.
+/// If this is uncommented, Dreamluau will be fully disabled.
 // #define DISABLE_DREAMLUAU
 
+/// If this is uncommented, /proc/icon_exists will attempt to load an initial cache from icon_exists_cache.json
+// #define PRELOAD_ICON_EXISTS_CACHE
```

Comment:
FWIW I wouldn't mind this being uncommented by default, if you think it'd be better.

Reply:
currently there's not even a build step to do this, i'm doing this manually / via TGS build scripts